### PR TITLE
css-two-column-layout - Jay Ghil

### PR DESCRIPTION
**Pull request by jghil**
css-two-column-layout

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    <div class="row">
+      <img src="images/software.svg" alt="software">
+      <div class="article">
+        <div class="column-half">
+          <h3>INTUITIVE, SMART SOFTWARE</h3>
+          <p>We build products that transform how healthcare facilities run across
+            Africa - making them radically more efficient operationally and
+            financially, a complete turnaround.</p>
+        </div>
+      </div>
```
Comment (BrettAlbright) [inline css-two-column-layout/index.html]: make sure that you have `column` elements wrapping the content. Dont forget your boxes!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good, make sure to have the column boxes in the rows before you add the content!

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
@jghil Please include a screenshot of what your page looks like after the most-recent update as a  comment on your pull request.

**Comment by jghil** [comment]
css-two-column-layout
Here is the my most recent update.  Sorry about that!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**